tools/deployment/serve/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# syntax = docker/dockerfile:experimental
```

Comment:
Move this dockerfile to the directory [here](https://github.com/open-mmlab/mmdetection/tree/master/docker) may help ease the management of dockers.

Reply:
Okey! I though about placing it there but didn't know if it was the right place.